src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.product-card {
+  box-sizing: border-box;
+  width: $card-width;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  color: $main-accent;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px;
+  transition: $transition-duration;
+
+  &__image-block {
+    margin: auto;
+  }
+
+  &__image {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
Since you've already set `display: block;` and `margin: auto;` for the &__image to center it horizontally, specifying `margin-left: auto;` and `margin-right: auto;` is redundant. You can safely remove these two lines.